### DISCUSSION TITLE
Pandora Window Is Completely Black, Preventing Launch (Steam Deck, Proton 9.0-3, MO2) [2.3.0 - Beta]

### DISCUSSION THREAD

**Discussion by AbstractParty** [Monitor221hz/Pandora-Behaviour-Engine-Plus#318] [Q&A]
I think this could be a bug, but I don't have a log (required) to submit it:

Using 2.3.0- Beta, Skyrim 1.5.97 on the Steam Deck with proton 9.0-3, and MO2

When launching Pandora from MO2 the window is completely black. No text, checkboxes, or button visible. Clicking randomly on the window (trying to hit the launch button) does not seem to do anything.

Note that the version I used previously (1.5.1 - Alpha) the Pandora window would flash continuously, but otherwise worked.

Note 2: I have no logs as I was not able to actually launch Pandora and generate logs.

Has anyone had success with using Pandora 2.3.0 - Beta on the Steam Deck (and if so, any suggestions what the issue might be)?

**Comment by AdamBrownawell** [comment]
I'm on Fedora 40 so there may be some differences but I've had this error occur about 50% the time that I launch Pandora through Mo2. I've found that resetting my Desktop Environment occasionally fixes it. Additionally, for me,  the buttons still work even if it's completely a black screen -- and clicking roughly center and down still produces a Pandora Output. Of course this may not be very helpful if you've been unable to check any patches yet.

**Reply by Jongbloed83** [reply]
Same problem, and more people have discussed it on the 'issues' tab. I downgraded to 2.15 which works fine.
